spec/app/models/mdm/host_spec.rb

```diff
@@ -1,4 +1,6 @@
 RSpec.describe Mdm::Host, type: :model do
+  include_context 'Rex::Text'
```

Comment:
I hear you and agree in general. In this case, I'll remove it if it would otherwise block this PR.